skewer/src/scene/scene.cc

```
@@ -75,12 +82,14 @@ void Scene::Build() {
     }
 
     for (uint32_t i = 0; i < (uint32_t)triangles_.size(); ++i) {
-        const Material& mat = materials_[triangles_[i].material_id];
-        if (mat.IsEmissive()) {
+        const Material* mat = (triangles_[i].material_id != kNullMaterialId)
+                                  ? &materials_[triangles_[i].material_id]
+                                  : nullptr;
+        if (mat->IsEmissive()) {
```

Cppcheck (2.20.0)

[warning] Possible null pointer dereference

(nullPointer)